src/layers/src/layer-utils.ts

```diff
@@ -79,6 +91,68 @@ export function getGeojsonLayerMetaFromArrow({
   };
 }
 
+/*
```

Comment:
Either cut or add comment explaining why we are keeping this code. Maybe this is related to the other PR?